types/stripe/index.d.ts

```diff
@@ -5,11 +5,12 @@
 //                 Amrit Kahlon <https://github.com/amritk/>
 //                 Adam Cmiel <https://github.com/adamcmiel>
 //                 Justin Leider <https://github.com/jleider>
+//                 Kamil Gałuszka <https://github.com/galuszkak>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
```

Comment:
Sure if thats the case that makes sense, however this also needs to work when loaded alongside v2 since stripe has not yet implemented everything in v3. When I load both, I get
```
Cannot redeclare block-scoped variable 'Stripe'.
```
But it works fine when exporting instead of declaring